dtschema/schemas/chosen.yaml

```diff
@@ -66,6 +66,17 @@ properties:
 
       This random value should be provided by bootloader.
 
+  arm,security-state:
+    $ref: types.yaml#/definitions/string
+    enum: [ non-secure, secure ]
```

Comment:
"non-secure" is the default for what's expected, so I don't think we should have a value for it. Then it becomes a question of whether we'd ever need something beyond "secure" and this could just be a boolean.

Reply:
From my experience, the default for ARMv7 is the kernel starting in secure mode, because there's no OP-TEE or TF-A that switches to normal world.